envs/vpn/conda/fedbiomed-node.yaml

```diff
@@ -10,56 +10,55 @@ channels:
 
 dependencies:
   # minimal environment
-  - python >=3.9,<3.10
-  - pip
-  - jupyter
-  - ipython
+  # python 3.11 recently released 2022-11 and not yet supported by some deps including: torchvision
+  - python >=3.10,<3.11
+  - pip >= 23.0
+  - jupyter ~=1.0.0
+  - ipython ~=8.13.2
   # tests
-  - pytest >6.2.2
-  - tinydb >=4.4.0,<5.0.0
-  - tabulate >=0.8.9,<0.9.0
+  - pytest ~=7.2.0
+  - pytest-cov ~=4.1.0
+  - tinydb ~=4.7.1
+  - tabulate >=0.9.0,<0.10.0
   # code
-  - GitPython >=3.1.14,<4.0.0
-  - requests >=2.25.1,<3.0.0
-  - paho-mqtt >=1.5.1,<2.0.0
-  - validators >=0.18.2,<0.19.0
-  - tqdm >=4.59.0,<5.0.0
-  - git
-  - packaging >=23.0,<24.0
+  - requests ~=2.29.0
+  - paho-mqtt ~=1.6.1
+  - validators >=0.20.0,<0.21.0
+  - tqdm ~=4.65.0
+  - git ~=2.40.1
+  - packaging ~=23.1
   # these two have to be aligned
-  - cryptography ~=39.0
-  - pyopenssl ~=23.0
+  - cryptography ~=40.0.0
+  - pyopenssl ~=23.1.1
   # git notebook striper
-  - nbstripout
-  - joblib >=1.0.1
-  # sklearn
-  #   scipy >= 1.9 from conda-forge needs recent GLIBC thus causes issue 389
-  #   with many current systems
-  #   another option is to install scipy from pip
-  - scipy >=1.8.0,<1.9.0
-  - scikit-learn >=1.0.0,<1.1.0
+  - nbstripout >=0.6.1,<0.7.0
```

Comment:
Hi @mvesin, 

As I understand `nbsripout` used for converting `.ipynb` files to `.py`  python modules. It is only used for interation test so it may not need to be installed on the node side. Or it can be part of `fedbiomed-ci` environment only (in the future)

Reply:
`nbstripout` is for removing notebooks' output (not for converting to `.py`).

This is coupled to some `.gitattributes` action for auto removing notebooks' output when comnitting notebooks. But it required some developers' configuration of its clone to be effective (and I can't find any documentation for it ...). Moreover, I don't know if someone is still using it. Asking all the developers. If not, we may completely remove it.

In the meantime:
- removed from node envs (we don't use notebooks in node envs)
- added comment "consider removing nbstripout" in yml